docs/contributing.md

```diff
@@ -27,6 +27,8 @@ These should all be run from within the project directory.
 - `yarn build`
   - Builds files from `/src` and outputs to `/lib` using webpack and UMD library target
   - `yarn build:watch` is also available
+- `yarn happo run`
+  - Starts Happo visual regression testing. Once the tests have finished running, a url will be provided to view the report. ** Note: Anyone can view the reports. To access happo to do anything beyond reading, you need to request access to the account or login with a `truss.works` domain email. **
```

Comment:
Should this command be included? The tests will general only be ran in CI.

Reply:
No, I think we should only list commands here that are meant to be used as part of the typical local development process. I think this can be removed.